content/specific-skill-success-criteria/classes-and-objects/_index.md

```diff
@@ -134,4 +134,54 @@ class Cuboid extends Rectangle {
 
 ### Overriding
 
-The learner should know that constructors and methods declared with final or static cannot be overridden. If a method cannot be inherited, it cannot be overridden.
\ No newline at end of file
+Learner should know that constructors and methods declared with final or static cannot be overridden. If a method cannot be inherited, it cannot be overridden.
+
+## Javascript
+
+### Constructors
+
+Learner should know how constructors work
+
+```
+class Rectangle {
+  constructor(length, width){
+    this.length = length
+    this.width = width
+  }
+}
+```
+### Instantiation
+
+Learner should know how to instantiate many objects from the same class and interact with them
+
+```
+const smallRectangle1 = new Rectangle(2, 4)
+const smallRectangle2 = new Rectangle(2, 4)
+const mediumRectangle = new Rectangle(4, 12)
+
+
+console.log(smallRectangle1.length) //2
+console.log(mediumRectangle.width) //12
+
+```
+
+### Inheritance and overriding
+
+Learner should understand inheritance and overriding and extending methods
+```
+class Rectangle {
+  ...
+  details(){
+    return `The length of the Rectangle is ${this.length}, the width of the Square is ${this.width}`;
```

Comment:
Should this not read the width of the Rectangle?

Reply:
Copy and paste is not healthy at all :(

Reply:
story of my life